CHANGELOG.rst

```diff
@@ -1,6 +1,6 @@
-====================
-Regolith Change Log
-====================
+=============
+Release Notes
+=============
 
 .. current developments
 
```

Comment:
Please add everything from the current changelog in here below the Release notes title

Reply:
Hi Professor, I have verified that the current version of CHANGELOG.rst has everything from the old repo's CHANGELOG, you can see it by expand down the files in Files changed

Reply:
ok, thanks.  Sorry for not seeing that.